packages/cli-doctor/src/tools/envinfo.ts

```diff
@@ -33,8 +33,8 @@ async function getEnvironmentInfo(
       System: ['OS', 'CPU', 'Memory', 'Shell'],
       Binaries: ['Node', 'Yarn', 'npm', 'Watchman'],
       IDEs: ['Xcode', 'Android Studio', 'Visual Studio'],
-      Managers: ['CocoaPods', 'RubyGems'],
-      Languages: ['Java'],
+      Managers: ['CocoaPods'],
+      Languages: ['Java', 'Ruby'],
```

Comment:
This change was also discussed in : https://github.com/react-native-community/cli/issues/1825

Whether we need `RubyGems` in Managers or `Ruby` in Languages

Reply:
Leaving this here to follow up: https://github.com/react-native-community/cli/pull/1818#discussion_r1097095344

There was some discussion about this in that thread. I think this tackles some of the issues raised by focusing on the Gemfile. Interestingly I'm not sure it's useful to consider the .ruby-version at all, even as a backup. 

## Tasks
- [ ] Can we build without a Gemfile (I don't think so)
- [ ] We could try fetch the actual requirement from the tagged releases, but I see lots of avenues for failure here (it's just tricky finding a way to grab the correct version for the matching version of RN - 2 network requests there at a minimum).

Reply:
> Can we build without a Gemfile (I don't think so)

We can but we would not have any guarantee on the version of Cocoapod used by the users. So, if they have an old version of Cocoapods installed in the current Ruby, they will be using that which may be not compatible with React Native.

So, I highly discourage that we don't use Gemfile.